js/client.js

```diff
@@ -835,12 +835,25 @@ function toId() {
 			// querySelector dates back to IE8 so we can use it
 			// fortunate, because form serialization is a HUGE MESS in older browsers
 			var elements = form.querySelectorAll('input[name], select[name], textarea[name], keygen[name]');
-			var out = [];
+			var out = [], checkboxes = {}, radios = {};
 			for (var i = 0; i < elements.length; i++) {
 				var element = elements[i];
-				// TODO: values are a mess in the DOM; checkboxes/select probably need special handling
```

Comment:
You deleted this comment but you don't seem to have handled the `select` element.